md/deadairfillv3.xml

```diff
@@ -139,8 +141,10 @@
 								<set_value name="$addWaresAmount" exact="0" />
 								<set_value name="$removeWaresAmount" exact="0" />
 								
-								<do_if value="$waresAmount gt $maxWaresAmount">
-									<set_value name="$removeWaresAmount" exact="($waresAmount - $maxWaresAmount)i" />

```

Comment:
This does not seem needed if shipyards and wharfs would have cargo decay logic implemented.
As I get it, this code makes sure to prevent situations when a resource takes more space than maximum cargo storage for that resource.